Analysis/Tasks/PWGHF/HFMCMatchingValidation.cxx

```diff
@@ -9,18 +9,23 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-// \file HFMCMAtchingValidation.cxx
-// \Monte Carlo Validation Code -- Gen e Reco level tasks
-// \author Antonio Palasciano <antonio.palasciano@cern.ch>, Università degli Studi di Bari & INFN, Sezione di Bari
+/// \file HFMCMAtchingValidation.cxx
+/// \MonteCarlo Validation Code -- Gen and Reco Level tasks
+/// \Gen Level: - Number of HF quarks produced  per collision
+///             - Number of D+->piKpi, D*+->piKpi, D0->K\pi, LambdaC->\pK\pi per collision
+///             - Momentum Conservation for these particles
+/// \Reco Level (only D0 matched candidates):
+///             - Gen-Reco Level Momentum Difference per component;
+///             - Gen-Reco Level Difference for secondary Vertex coordinates and decay length;
```

Comment:
These are still not [Doxygen commands](https://www.doxygen.nl/manual/commands.html).